packages/trader/src/AppV2/Components/TradeParameters/Stake/stake.tsx

```diff
@@ -15,23 +23,200 @@ const BASIS = {
     STAKE: 'stake',
 };
 
+const getSortedIndex = (type: string, index?: number) => {
+    switch (getContractTypePosition(type as 'CALL')) {
```

Comment:
Same in packages/trader/src/AppV2/Components/PurchaseButton/purchase-button.tsx

Reply:
PR is not ready for review) it's not how I'm gonna leave it) don't worry)